docassemble/AssemblyLine/al_general.py

```diff
@@ -87,7 +88,21 @@ def address_fields(
         `users[0].address.address`.
         Optionally, add a `show if` modifier to each field. The `show if` modifier
         will not be used if the `allow_no_address` modifier is used.
+
```

Comment:
Should also mention what country codes are valid (maybe a link to https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements).

```suggestion
    `country_code` should be an ISO-3166-1 alpha-2 code (https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements)
    
```